README.md

```diff
@@ -1,5 +1,39 @@
 # xeus-calc
 
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuantStack/xeus-calc/master)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/QuantStack/xeus-calc/stable?filepath=notebooks/Xeus-calc.ipynb)
 
 Calculator as a Jupyter Kernel implemented with Xeus
+
+## Usage
+
+Launch the Jupyter notebook with `jupyter notebook` or Jupyter lab with `jupyter lab` and launch a new Python notebook by selecting the **xeus-calc** kernel.
```

Comment:
and launch a new Python notebook => and start a new Python notebook

Reply:
It is not Python though...

Reply:
Indeed. That should be changed and the kernelspec should be updated accordingly.

Reply:
Yes, I will change that.